### PR TITLE
Seed page and dropdown improvements

### PR DESCRIPTION
**Pull request by cloudhead**

### CODE REVIEW

**Review by sebastinez — APPROVED** [review]
One small comment on returning to the `head` branch in user and org context.
And in `src/base/seeds/View.svelte` L:20 IMO we should change width to 720px equal to Org and User View page.
If you want I can provide a commit for that.

File: src/base/projects/BranchSelector.svelte [hunk 1]
```diff
@@ -61,75 +62,43 @@
...
-        {branchLabel}
-      </div>
       <div
-        class="dropdown branch-dropdown"
-        class:branch-dropdown-without-label={!isLabel}
-        class:branch-dropdown-visible={branchesDropdown}
+        class="stat branch"
+        class:not-allowed={!showSelector}
+        class:hidden={!isLabel}
+        on:click={() => showSelector && toggleDropdown("branch")}
```
[the remark this answers is not in the document]
Comment (cloudhead) [reply src/base/projects/BranchSelector.svelte:83]: Doesn't clicking on the anchor do that already for you? Or you mean something else?

[the remark this answers is not in the document]
Comment (sebastinez) [reply src/base/projects/BranchSelector.svelte:83]: Yup we can leave it with the anchor for now. Not sure how intuitive this is, maybe in the future we can have a different way.
For example a disable filter "X" that navigates back to head, similar to the peer selector idea.

[the remark this answers is not in the document]
Comment (cloudhead) [reply src/base/projects/BranchSelector.svelte:83]: Ah yes, that's a good idea!

**Pull request merged**